src/modules/tesla-android/filesystem/etc/network/interfaces

```diff
@@ -8,10 +8,14 @@ auto wlan0
 allow-hotplug wlan0
 iface wlan0 inet manual
 
+auto eth1
+allow-hotplug eth1
+iface eth1 inet dhcp
+
 auto br0
 iface br0 inet static
 	address 3.3.3.1
 	netmask 255.255.255.0
-	bridge_ports eth0 wlan0
+	bridge_ports eth0 wlan0 eth1
```

Comment:
Same thing as before, if eth1 is a source there is no need to add it to a bridge that handles dhcp for the network. I suppose